libs/vkd3d/resource.c

```diff
@@ -4004,6 +4004,20 @@ static void vkd3d_buffer_view_get_bound_range_ssbo(struct d3d12_desc *descriptor
     }
 }
 
+static bool vkd3d_buffer_view_get_bound_range_typed(struct d3d12_desc *descriptor,
```

Comment:
I'm not a big fan of the name here since it does something different from `get_bound_range_ssbo` - this creates an actual view object, the other one really just computes some numbers.

Reply:
Fixed.